### DISCUSSION TITLE
Striping in Landsat mosaics

### DISCUSSION THREAD

**Discussion by mennostraatsma** [microsoft/PlanetaryComputer#111] [Q&A] [answered]
Landsat images require mosaicking to cover areas larger than the width of a single scene. However, in the 'Explorer' as well as in scripted output missing values are generated in stripes or wedges. The missing values depend on the location and zoom level and occur also in Landsat8, hence unrelated to to Landsat7's malfunctioning sensor.

For example, this querie
https://planetarycomputer.microsoft.com/explore?c=5.6177%2C52.0152&z=9.35&v=2&d=landsat-c2-l2&s=false%3A%3A100%3A%3Atrue&ae=0&m=Most+recent+%28low+cloud%29&r=Natural+color 
gives
![image](https://user-images.githubusercontent.com/29101043/193615186-b3851421-9e6b-40ce-b81f-509f7b6b3f12.png)

What causes the striping and what would be the solution?

**Comment by TomAugspurger** [comment, marked as answer]
> also in Landsat8, hence unrelated to to Landsat7's malfunctioning sensor.

The link you shared is for the STAC collection `landsat-c2-l2`, which does include data from Landsat 7.  If you go into the "Advanced" filters, you can filter the "Platform" to be landsat-8 and landsat-9


<img width="962" alt="image" src="https://user-images.githubusercontent.com/1312546/193632282-903795db-81ab-4e24-a647-42647b7b06ce.png">


https://planetarycomputer.microsoft.com/explore?c=5.6177%2C52.0155&z=9.35&v=2&d=landsat-c2-l2&m=cql%3A6f33cb4aa18dee0a5418bc8fe56c1879&r=Natural+color&s=false%3A%3A100%3A%3Atrue&ae=0

**Reply by mennostraatsma** [reply]
Thanks, I assumed Landsat 7 only had linear stripes of missing data, not the wedge-shaped ones.
